src/opencl/utils.rs

```diff
@@ -1,7 +1,7 @@
 use std::convert::TryFrom;
 
 use log::{debug, warn};
-use opencl3::device::CL_UUID_SIZE_KHR;
+pub use opencl_sys::CL_UUID_SIZE_KHR;
```

Comment:
Why not keeping the current import?

Reply:
CL_UUID_SIZE_KHR was no longer public from opencl3, so I had to include it from opencl_sys where opencl3 also imports it from.

Reply:
I've opened an upstream fix for that https://github.com/kenba/opencl3/pull/54.

Reply:
It was merged and 0.9.1 was released. So if you update `opencl3` to `0.9.1` in the `Cargo.toml you should be able to use `opencl3::device::CL_UUID_SIZE_KHR` again.